src/pages/operations/OperationList.tsx

```diff
@@ -84,6 +79,9 @@ const OperationList: FC = () => {
               <div className="u-truncate u-text--muted">
                 <OperationInstanceName operation={operation} />
               </div>
+              <div className="u-text--muted">
```

Comment:
```suggestion
              <div className="u-text--muted u-truncate" title={getProjectName(operation)}>
```
Better be prepared for users with super long project names not overflowing into the next line. While still making the full name visible on hover with the title.